tests/test_handler_stats.py

```diff
@@ -136,18 +136,19 @@ def _train_func(engine, batch):
             # set up testing handler
             stats_handler = StatsHandler(name=key_to_handler, tag_name=key_to_print, logger_handler=handler)
             stats_handler.attach(engine)
+            stats_handler.logger.setLevel(logging.INFO)
 
             engine.run(range(3), max_epochs=2)
             handler.close()
             stats_handler.logger.removeHandler(handler)
             with open(filename) as f:
                 output_str = f.read()
```

Comment:
What was the issue with this test? I remember previously it worked

Reply:
Hi @wyli ,

Sorry I forgot to describe the issue here.
The previous test code actually didn't run into the `assert`:
```py
for idx, line in enumerate(output_str.split("\n")):
    if grep.match(line):  # ---------------> here is FALSE, so following logic didn't run at all and passed
        if idx in [1, 2, 3, 6, 7, 8]:
            self.assertTrue(has_key_word.match(line))
```

Thanks.

Reply:
Right, looks like we have the same issue for the other test cases https://app.codecov.io/gh/Project-MONAI/MONAI/blob/dev/tests/test_handler_stats.py

Reply:
Hi @wyli ,

Nice catch by the tool! Let me update them all.

Thanks.

Reply:
Updated all the wrong test cases in the latest commit.

Thanks.